src/server/api/geo/getForestAgreementLayer.ts

```diff
@@ -8,16 +8,21 @@ import Requests from '@server/utils/requests'
 
 export const getForestAgreementLayer = async (
   req: Request<
-    { countryIso: CountryIso },
     never,
     never,
-    { gteAgreementLevel: number; gteHansenTreeCoverPerc: number; layer: Array<ForestSource>; opacity: number }
+    never,
+    any & {
```

Comment:
why do you extend any here?

Reply:
I just checked out this branch, and I see no error if I remove any .
<img width="817" alt="Screenshot 2022-10-25 at 17 01 29" src="https://user-images.githubusercontent.com/1138156/197809862-5854a575-b904-4cc5-855e-ecd723f34487.png">

Reply:
I am changing all parameters to string as ParsedQs interface allows only string as a type. In future PR I will create a middlware function to validate number parameters.